bors.toml

```diff
@@ -0,0 +1,4 @@
+status = [
+  "ci/gitlab/%",
+]
+delete_merged_branches = true
```

Comment:
Does this delete the merged branch from the fork, or does it delete the temporary merge between the PR branch and master?

Reply:
I had to check bors's source for this, but it seems this removes branches inside the current repo after they are merged as part of a PR. It does not / cannot delete branches of forks.